frontend/src/components/header.js

```diff
@@ -21,6 +21,14 @@ BootstrapNavLink.propTypes = {
     to: PropTypes.string,
 };
 
+function BootstrapNavItem(props) {
```

Comment:
Is this needed? Also, all components should have a doc string, even if it's just `A wrapper for BootstrapNavLink so that the item fits in a nav bar`, or something like that...

Reply:
also, the `BootstrapNavLink` component isn't used in any other files. In fact, it is redefined in `frontend/src/views/dev_frame/index.js`, so I'm not sure why this function is exported at all. I will rework it and `BootstrapNavItem` into one function.